.github/workflows/build.yml

```
@@ -30,7 +30,7 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
       - name: Install Tox and any other packages
-        run: pip install tox
+        run: pip install .[dev]
```

```suggestion
        run: pip install tox
```

Keep tox here, it installs all it needs in a virtual env